src/pandora/service.ts

```diff
@@ -557,8 +560,9 @@ export class PandoraService {
     const selectedCosts = withCDN ? costs.withCDN : costs
     const rateNeeded = selectedCosts.perEpoch
 
-    // Default lockup period is 10 days = 28,800 epochs
-    const lockupNeeded = rateNeeded * TIME_CONSTANTS.DEFAULT_LOCKUP_PERIOD
+    // Calculate lockup period based on provided days (default: 10)
+    const lockupPeriod = BigInt(lockupDays ?? 10) * TIME_CONSTANTS.EPOCHS_PER_DAY
```

Comment:
```suggestion
    const lockupPeriod = BigInt(lockupDays ?? TIME_CONSTANTS.DEFAULT_LOCKUP_PERIOD) * TIME_CONSTANTS.EPOCHS_PER_DAY
```

just try and avoid magic numbers where possible, makes it easier, it does mean that our comments can get out of sync with implementation but it's better than having code out of sync with itself

Reply:
I guess `DEFAULT_LOCKUP_PERIOD` is in epochs? feel free to change the constant's name to `DEFAULT_LOCKUP_DAYS`, set it to `10`, and then you can multiply by epochs